database/procedures/author/create.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE PROCEDURE CREATE_AUTHOR(
+    country_id INT,
+    firstname CHAR(50),
+    lastname CHAR(50)
+)
+LANGUAGE plpgsql    
+AS $$
+BEGIN
+    INSERT INTO AUTHOR(COUNTRY_ID, FIRSTNAME, LASTNAME)
+    VALUES(country_id, firstname, lastname);
+    COMMIT;
```

Comment:
Why do you use COMMIT here?

Reply:
Usualy functions that create new entities shoud return added entity id

Reply:
addressed